lurk/riscv64.patch

```diff
@@ -1,38 +1,33 @@
 --- PKGBUILD
 +++ PKGBUILD
-@@ -9,16 +9,31 @@ arch=('x86_64')
+@@ -9,13 +9,26 @@ arch=('x86_64')
  url="https://github.com/JakWai01/lurk"
  license=('MIT' 'Apache')
  depends=('gcc-libs')
 -makedepends=('cargo')
+-source=("$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz")
+-sha512sums=('f545b83f5f6fc84399419394c606e3d7c9c4a5ed094ae171f4f226768609ee7c1d364d13f862efd6aa2d21615cd7c14165dc3c4c4a2b2f9148fba4963b62f401')
 +makedepends=('cargo' 'git')
- source=("$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz"
--        "$pkgname-$pkgver-fix-tests.patch::$url/commit/87eb4aa8bf9a551b24cec2146699cb2c22d62019.patch")
-+        "$pkgname-$pkgver-fix-tests.patch::$url/commit/87eb4aa8bf9a551b24cec2146699cb2c22d62019.patch"
-+        "$pkgname-add-riscv64-support.patch::https://github.com/hack3ric/lurk/commit/44cfa7df0caaae60d5a93ec52041f2da35612b7d.diff"
-+        "git+https://github.com/nix-rust/nix#tag=v0.26.2"
++source=("$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz"
++        "add-riscv64-support.patch::https://github.com/JakWai01/lurk/commit/ac8aa8e548570ec419ffe55106503e1c04d4a4bc.diff"
```

Comment:
Is this upstreamed? It shows out of repo and I cannot find a relative PR.

Reply:
No. I'm still waiting for https://github.com/nix-rust/nix/pull/2044 to get merged.